pom.xml

```diff
@@ -75,37 +75,37 @@
         <kotlin.compiler.incremental>true</kotlin.compiler.incremental>
 
         <!-- plugins -->
-        <maven.gpg.version>1.6</maven.gpg.version>
-        <sonatype.nexus.staging.version>1.6.8</sonatype.nexus.staging.version>
-        <maven.compiler.version>3.8.0</maven.compiler.version>
+        <maven.gpg.version>3.0.1</maven.gpg.version>
+        <sonatype.nexus.staging.version>1.6.13</sonatype.nexus.staging.version>
+        <maven.compiler.version>3.10.1</maven.compiler.version>
         <maven.pmd.version>3.15.0</maven.pmd.version>
-        <maven.license.version>3.0</maven.license.version>
-        <maven.source.version>3.2.0</maven.source.version>
-        <maven.jar.version>3.2.0</maven.jar.version>
-        <maven.shade.version>3.2.2</maven.shade.version>
-        <jacoco.version>0.8.7</jacoco.version>
-        <maven.surefire.version>3.0.0-M4</maven.surefire.version>
-        <build.helper.version>3.0.0</build.helper.version>
+        <maven.license.version>4.1</maven.license.version>
+        <maven.source.version>3.2.1</maven.source.version>
+        <maven.jar.version>3.2.2</maven.jar.version>
+        <maven.shade.version>3.3.0</maven.shade.version>
+        <jacoco.version>0.8.8</jacoco.version>
+        <maven.surefire.version>3.0.0-M7</maven.surefire.version>
+        <build.helper.version>3.3.0</build.helper.version>
         <maven.templating.version>1.0.0</maven.templating.version>
 
         <!-- dependencies -->
-        <jfx.version>18</jfx.version>
-        <kotlin.version>1.6.0</kotlin.version>
-        <attach.version>4.0.13</attach.version>
-        <jackson.version>2.12.1</jackson.version>
+        <jfx.version>19-ea+10</jfx.version>
+        <kotlin.version>1.7.10</kotlin.version>
+        <attach.version>4.0.15</attach.version>
+        <jackson.version>2.13.3</jackson.version>
 
         <!-- test dependencies -->
-        <junit.jupiter.version>5.8.2</junit.jupiter.version>
-        <junit.platform.version>1.8.2</junit.platform.version>
-        <hamcrest.version>1.3</hamcrest.version>
+        <junit.jupiter.version>5.9.0</junit.jupiter.version>
+        <junit.platform.version>1.9.0</junit.platform.version>
+        <hamcrest.version>2.2</hamcrest.version>
     </properties>
 
     <dependencies>
         <dependency>
             <groupId>org.jetbrains.kotlin</groupId>
             <artifactId>kotlin-stdlib</artifactId>
             <version>${kotlin.version}</version>
-            <classifier>modular</classifier>
+<!--            <classifier>modular</classifier>-->
```

Comment:
Given there is no classifier now, we can delete this line.

Reply:
deleted